tests/extras/datasets/spark/test_spark_dataset.py

```diff
@@ -192,6 +208,31 @@ def test_load_options_csv(self, tmp_path, sample_pandas_df):
         spark_df = spark_data_set.load()
         assert spark_df.filter(col("Name") == "Alex").count() == 1
 
+    def test_load_options_schema(
+        self, tmp_path, sample_pandas_df, sample_spark_df_schema
+    ):
+        filepath = (tmp_path / "data").as_posix()
+        schemapath = (tmp_path / "schema.json").as_posix()
+
+        # Write CSV dataset
+        local_csv_data_set = CSVDataSet(filepath=filepath)
+        local_csv_data_set.save(sample_pandas_df)
+
+        # Write schema
+        # TODO: Should this be tackled in another way?
```

Comment:
Not sure what you meant by this question, this looks fine to me. I would just use the pathlib api instead (st like `schemapath.write_text()`) if I could, but otherwise I'm not offended by this.

Reply:
The question was more about the general test setup, I must admit that I had never used PyTest before.

Reply:
Done!